kornia/feature/laf.py

```diff
@@ -494,12 +488,12 @@ def extract_patches_from_pyramid(
             if (scale_mask.float().sum()) == 0:
                 continue
             scale_mask = (scale_mask > 0).view(-1)
-            grid = generate_patch_grid_from_normalized_LAF(cur_img[i:i + 1], nlaf[i:i + 1, scale_mask, :, :], PS)
+            grid = generate_patch_grid_from_normalized_LAF(cur_img[i : i + 1], nlaf[i : i + 1, scale_mask, :, :], PS)
```

Comment:
is this fine with Flake8?

Reply:
yep, that's the `E203` -- I made sure to follow the readme that black provides

(by the way, I maintain flake8)